core/src/main/scala/org/bykn/bosatsu/TypedExpr.scala

```diff
@@ -139,6 +138,93 @@ object TypedExpr {
     }
   }
 
+  implicit val traverseTypedExpr: Traverse[TypedExpr] = new Traverse[TypedExpr] {
```

Comment:
sorry, I think I caused a merge conflict here... I added `Functor[TypedExpr]` which I needed, but traverse is a super set of that, so if you merge master, you can do so by either removing my `Functor` or copying the `def map` method into your Traverse with a `override def map` (it is *slightly* more efficient to manually implement map than do the default implementation which uses `traverse` with `Id[A]`

Reply:
oh nice, easy enough